### ISSUE TITLE
Feature Request: Automatically generating documentation

### ISSUE THREAD

**Issue by tqma113** [Lucifier129/farrow#83]
In actual business development, there is a high demand for automatically generating documents

**Comment by Lucifier129** [comment]
The codegen result of `farrow-api` is useful as documentation

```typescript
class MyApiInput extends ObjectType {
  name = {
    description: 'the text in here will be convert to a comment for api consumer',
    deprecated: 'deprecated info',
    [Type]: String
}
```

It may be also useful to render the info to a web page.

**Comment by tqma113** [comment]
Yes, It has the ability. But for the business development, it require more complete generation capability. Like GraphQL playground or another form.

**Comment by tqma113** [comment]
Best practices for API version management are also needed.

**Comment by Lucifier129** [comment]
It will be easy use implement a documentation generator via the introspection of `farrow-api`, render the JSON protocol to a pretty SPA page.

Are you interested in this direction? PR is welcome.
